2Peers-Backend/models/Teacher.js

```diff
@@ -1,10 +1,25 @@
 const db = require('../database/db');
 
 class Teacher {
+  static signUp(body) {
+    const queryTeacher = 'INSERT INTO teachers (name, email, profilepic, encryptedpassword, archived) VALUES ($1, $2, $3, $4, $5)';
```

Comment:
You added a checkbox to the teacher table (one of your init.sql changes) but you're excluding it here. Was the checkbox unnecessary?